ui/app/services/auth.js

```diff
@@ -296,21 +306,16 @@ export default Service.extend({
       resp.policies
     );
 
-    if (resp.renewable) {
-      Object.assign(data, this.calculateExpiration(resp));
-    } else if (resp.type === 'batch') {
-      // if it's a batch token, it's not renewable but has an expire time
-      // so manually set tokenExpirationEpoch and allow expiration
-      data.tokenExpirationEpoch = new Date(resp.expire_time).getTime();
-      this.set('allowExpiration', true);
```

Comment:
this was getting overridden by line 312 anyway, so seems like we don't need it?

Reply:
I think instead we should set it to the value of `!resp.renewable` -- otherwise i believe it will attempt to refresh the token which will fail